packages/types/src/extrinsic/signedExtensions/encointer.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017-2022 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ExtDef } from './types';
+
+export const encointer: ExtDef = {
+  ChargeAssetTxPayment: {
+    extrinsic: {
+      tip: 'Compact<Balance>',
+      // eslint-disable-next-line sort-keys
+      assetId: 'Option<CommunityIdentifier>'
+    },
```

Comment:
Here I am uncertain if it matters that this type does not exist when we connect to a chain that does not have this type defined.